Mathlib/Data/Set/Basic.lean

```diff
@@ -2168,4 +2168,27 @@ end Disjoint
 @[simp] theorem Prop.compl_singleton (p : Prop) : ({p}ᶜ : Set Prop) = {¬p} :=
   ext fun q ↦ by simpa [@Iff.comm q] using not_iff
 
+namespace Equiv
+
+/-- Given a predicate `p : α → Prop`, produces an equivalence between
+  `Set {a : α // p a}` and `{s : Set α // ∀ a ∈ s, p a}`. -/
+protected def setSubtypeComm {α : Type*} (p : α → Prop) :
```

Comment:
There is already `variable {α : Type u}` ... can you use it?

Reply:
@madvorak Sure!